src/evaluation/javascript.ts

```diff
@@ -1,13 +1,15 @@
 import type { ReplyMessageOptions } from 'discord.js';
 import { spawn } from 'child_process';
+import { inspect } from 'util';
 import { BaseEvaluationSystem } from './base';
 
 export class JavaScriptEvaluationSystem extends BaseEvaluationSystem {
     public evaluate(content: string): Promise<ReplyMessageOptions> {
         return new Promise(res => {
             const child = spawn('node',
                 [
-                    '-i',
+                    '-e',
+                    'repl.start({ ignoreUndefined: false, useGlobal: true, breakEvalOnSigint: true, prompt: "" })',
```

Comment:
```suggestion
                    'repl.start({ useGlobal: true, breakEvalOnSigint: true, prompt: "" })',
```

`ignoreUndefined` はデフォルトで `false` のため